typing/src/main/java/org/navgurukul/typingguru/keyboard/dialog/KeyboardDialogFragment.kt

```diff
@@ -1,6 +1,7 @@
 package org.navgurukul.typingguru.keyboard.dialog
 
 import android.app.Dialog
+import android.content.DialogInterface
```

Comment:
Why are these changes in this PR?

Reply:
We had that bug for the dismiss of activity. It was a single line so I pushed with this.